### PR TITLE
Update systemd unit file

### PR DESCRIPTION
**Pull request by smarlowucf**
Billing adapter needs network and uyuni service running

### CODE REVIEW

**Comment by smarlowucf** [comment]
This is good for SUMA. But since it's in the generic adapter repo do we want to drop the uyuni requirement? That's very specific to SUMA.

File: systemd/csp-billing-adapter.service [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=CSP usage reporting
-Wants=billing-data-service.service
-After=billing-data-service.service
+Wants=network.target billing-data-service.service
```
Comment (rjschwei) [inline systemd/csp-billing-adapter.service:3]: This is not correct, we require the network and want the billing data service, i.e. this should be:
```
Requires=network.target
Wants=billing-data-service.service
```

The service cannot run if there is no network.

**Review by rjschwei — CHANGES_REQUESTED** [review]

File: systemd/csp-billing-adapter.service (hunk 1 above)
Comment (smarlowucf) [reply systemd/csp-billing-adapter.service]: Rebased with these changes

File: systemd/csp-billing-adapter.service [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=CSP usage reporting
+Requires=network.target
 Wants=billing-data-service.service
-After=billing-data-service.service
```
Comment (rjschwei) [inline systemd/csp-billing-adapter.service:4]: Sorry, obviously my previous comment was confusing. The `After` settings were correct. Without `After` this service will be started at the same time that the other services and as such a race condition exists.

**Review by rjschwei — CHANGES_REQUESTED** [review]

File: systemd/csp-billing-adapter.service (hunk 2 above)
Comment (smarlowucf) [reply systemd/csp-billing-adapter.service:4]: Okay, force pushed again with the original After.

**Review by rjschwei — APPROVED** [review]

**Review by jesusbv — APPROVED** [review]

**Pull request merged**